terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -62,7 +62,7 @@ locals {
 
   bcl_convert_wfl_version = {
     dev  = "3.7.5"
-    prod = "3.7.5--fade666"
+    prod = "3.7.5--f1e67a3"
```

Comment:
This looks like it has regressed from fade666 (which was on 10 Feb 2022) to f1e67a3 (which was on 7 June 2021)